package.json

```diff
@@ -18,9 +17,9 @@
     "@tailwindcss/forms": "^0.5.3",
     "@tailwindcss/typography": "^0.5.8",
     "autoprefixer": "^10.4.14",
-    "cable_ready": "^5.0.0",
+    "cable_ready": "5.0.0-pre8",
```

Comment:
I think we were at `-pre9` prior to this automated upgrade. Not a blocker I don't think.

Reply:
No problem, I just use pre8 since that's what we currently have on the main branch

https://github.com/bullet-train-co/bullet_train/blob/dacdcc913cf68d93d685db3897c4180e70804e51/package.json#L21

Reply:
Oh, then all good with pre8